proposals/handler.go

```diff
@@ -642,13 +647,14 @@ func (h *Handler) checkVotesConsistency(ctx context.Context, b *types.Ballot) er
 		if voted, ok := layers[vote.LayerID]; ok {
 			// already voted for a block in this layer
 			if voted != vote.ID && vote.LayerID.Add(h.cfg.Hdist).After(b.Layer) {
-				h.logger.WithContext(ctx).With().Warning("ballot doubly voted within hdist, set smesher malicious",
-					b.ID(),
-					b.Layer,
-					log.Stringer("smesher", b.SmesherID),
-					log.Stringer("voted_bid", voted),
-					log.Stringer("voted_bid", vote.ID),
-					log.Uint32("hdist", h.cfg.Hdist),
+				h.logger.Warn("ballot doubly voted within hdist, set smesher malicious",
+					log.ZContext(ctx),
+					zap.Stringer("ballot", b.ID()),
+					b.Layer.Field().Zap(),
```

Comment:
This probably needs to be changed either in this or the other PR

Reply:
changed